configurations/default/env.yml.tmp

```diff
@@ -7,5 +7,6 @@ MAPBOX_ATTRIBUTION: <a href="https://www.mapbox.com/about/maps/" target="_blank"
 SLACK_CHANNEL: optional-slack-channel
 SLACK_WEBHOOK: optional-slack-webhook
 GRAPH_HOPPER_KEY: your-graph-hopper-key
+GRAPH_HOPPER_URL: https://graphhopper.com/api/1/
```

Comment:
This should be commented out and contain an example of a graph hopper instance URL (perhaps `http://localhost:8989/api/` if that's the default API location for graph hopper).